forumMessages/migrations/0002_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.4 on 2020-12-11 12:38
+
+from django.db import migrations, models, transaction
+from django.utils import timezone
+from django.contrib.auth.models import User
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('forumMessages', '0001_initial.py'),
```

Comment:
This should be '0001_initial' without the .py
See https://docs.djangoproject.com/en/3.1/topics/migrations/#migration-files